src/modules/scene/add_question_scene.py

```diff
@@ -32,9 +36,14 @@ def start_scene(self):
             for event in pg.event.get():
                 self.handle_quit(event)
 
-
                 self.__question_prompt.handle_event(event)
-                _ = [p.handle_event(event) for p in self.__option_prompts]
+                # Handle options + enforce single correct answer
+                for i, p in enumerate(self.__option_prompts):
+                    if p.handle_event(event):
+                        for j, p in enumerate(self.__option_prompts):
+                            if i != j:
```

Comment:
Any reasons why we don't support multiple correct answers?

Reply:
oh i just assumed currently we only have one type of question: MultipleChoice (which means only one correct answer?